notebooks/tr/agent_data_analyst.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>Ajan desek daha doğru olur, bot diyince chatbot çağrışımı yapıyor, bir de agent yerine ajan yazabiliriz kalan her yerde tutarlılık adına</p><p>Dataframelerden -&gt; dataframe'lerden*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Ajan olarak bilerek çok çevirmek istemedim biraz tuhaf geldi açıkçası assistan olarak düşündüm ama acaba her yerde agent olarak mı bıraksak?

Reply:
ajan en doğrusu bence tam altını doldurduğu için
asistanın kapsamı daha geniş